README.md

```diff
@@ -5,57 +5,41 @@
 [![GitHub last commit](https://img.shields.io/github/last-commit/pylhc/sdds.svg?style=popout)](https://github.com/pylhc/sdds/)
 [![GitHub release](https://img.shields.io/github/release/pylhc/sdds.svg?style=popout)](https://github.com/pylhc/sdds/)
 
-This package provides reading and writing functionality for **self describing data sets (sdds)**. 
-
-## Getting Started
-
-### Prerequisites
-
-The only third-party package sdds depends on is `numpy`.
+## Description
 
+This package provides reading and writing functionality for [**self describing data sets (sdds)**](https://ops.aps.anl.gov/SDDSIntroTalk/slides.html) files.
+On the python side, the data is stored in a class structure with attributes corresponding to the sdds-format itself (see [sdds-format](https://ops.aps.anl.gov/manuals/SDDStoolkit/SDDStoolkitsu2.html)). 
 
-### Installing
+## Getting Started
 
 Installation is easily done via `pip`. The package is then used as `sdds`.
 
 ```
 pip install sdds
 ```
 
-Example:
+Example use:
 
-```
+```python
 import sdds
 
-sdds_data = sdds.read('path_to_input.sdds')
-sdds.write(sdds_data, 'path_to_output.sdds')
+sdds_data = sdds.read("path_to_input.sdds")
+sdds.write(sdds_data, "path_to_output.sdds")
 ```
 
-
-## Description
-
-Reading and writing capabilities for [sdds-files](https://ops.aps.anl.gov/SDDSIntroTalk/slides.html)
-are provided by this package. On the python side, the data is stored in a class structure
-with attributes corresponding to the sdds-format itself 
-(see [sdds-format](https://ops.aps.anl.gov/manuals/SDDStoolkit/SDDStoolkitsu2.html)).
-
-
 ## Known Issues
 
-- Can't read ASCII files
```

Comment:
I was lazy to fix this for quite a while, nice

Reply:
:)